ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableArray.java

```diff
@@ -25,4 +27,6 @@
   ReadableMap getMap(int index);
   Dynamic getDynamic(int index);
   ReadableType getType(int index);
+  ArrayList<Object> toArrayList();
```

Comment:
Use the interface instead of the concrete type, so `List` instead of `ArrayList`